editor/src/clj/editor/html_view.clj

```diff
@@ -247,7 +247,7 @@
   (if-let [new-resource-node
            (when (string/starts-with? new-location (http-server/url (get-http-server! project)))
              (let [resource-path (subs new-location (count (http-server/url (get-http-server! project))))
-                   resource-node (project/get-resource-node project resource-path)
+                   resource-node (project/get-resource-node project (URLDecoder/decode resource-path))
```

Comment:
Fixes a bug that throws an error when the file name of the asset contains special characters like spaces (we get the url by listening to the location of the web-engine).